mmv1/templates/terraform/resource.erb

```diff
@@ -287,7 +288,7 @@ func resource<%= resource_name -%>Create(d *schema.ResourceData, meta interface{
 <% unless object.taint_resource_on_failed_create -%>
         // The resource didn't actually create
         d.SetId("")
-<%  end -%>         
+<%  end -%>
```

Comment:
huh! wouldn't have thought this would cause a diff, since the marker at the end indicates we should eat whitespace. Weird!

Reply:
fixed